alcs-frontend/src/app/services/compliance-and-enforcement/compliance-and-enforcement.service.ts

```diff
@@ -25,10 +25,11 @@ export class ComplianceAndEnforcementService {
   async create(
     updateDto: UpdateComplianceAndEnforcementDto,
     createInitialSubmitter = false,
+    createInitialProperty = false,
   ): Promise<ComplianceAndEnforcementDto> {
     return await firstValueFrom(
       this.http.post<ComplianceAndEnforcementDto>(
-        `${this.url}?createInitialSubmitter=${createInitialSubmitter}`,
+        `${this.url}?createInitialSubmitter=${createInitialSubmitter}&createInitialProperty=${createInitialProperty}`,
```

Comment:
100% up to you if you want to bother changing this. This works fine as is, and I obviously did it this way to start, but I recently discovered what I think is a bit nicer way to handle this. Here's what I did for submission docs:

```ts
let params = new HttpParams();

if (fileNumber) {
  params = params.set('fileNumber', fileNumber);
}
for (const typeCode of typeCodes) {
  params = params.append('typeCodes', typeCode);
}

return firstValueFrom(this.http.get<ComplianceAndEnforcementDocumentDto[]>(this.url, { params }));
```